aspnetcore/security/authentication/individual.md

```diff
@@ -32,6 +32,17 @@ dotnet new razor -au Individual
 
 ::: moniker-end
 
+## No Authentication
+
+Authentication is specified in the .NET Core CLI with `-au` option. In Visual Studio, the **Change Authentication** dialog is available for new web applications. The default for new web apps in Visual Studio is **No Authentication**.
+
+Projects created with no authentication:
+
+* Don't contain web pages and UI to sign in and sign out.
```

Comment:
It's a bit unfortunate, but these ended up as a pair of *double negatives* coming off of "with no/without" above. It might be a bit better if instead of "Don't contain ..." it said "Lack ..." ?? maybe